test/parallel/test-http-outgoing-proto.js

```diff
@@ -93,7 +93,7 @@ assert.throws(() => {
   code: 'ERR_INVALID_ARG_TYPE',
   name: 'TypeError',
   message: 'The first argument must be of type string or an instance of ' +
-           'Buffer. Received type number (1)'
+           'Buffer or Uint8Array. Received type number (1)'
```

Comment:
Is changing this message potentially breaking? ping @ronag @nodejs/lts

Reply:
It has an error code, so our policy doesn’t consider it a breaking change, and it’s a typechecking error, so my common sense also says that this should never be an issue.

Reply:
Okay, thanks 👍.  FWIW, this was a query during our release shadowing/onboarding session today. I'll add a link to the [Deprecation Policy](https://github.com/nodejs/node/blob/master/doc/guides/collaborator-guide.md#breaking-changes) to the Release guide.